postlammps.f90

```diff
@@ -350,14 +350,16 @@ end subroutine Block_Analysis
 
   !=================================================================================================
 
-  subroutine Print_Properties( np, property, npoints, value, interval )
+  subroutine Print_Properties( np, property, npoints, value, interval)
     integer,       intent(in) :: np, npoints
     character(sl), intent(in) :: property(np)
     real(rb),      intent(in) :: value(np,npoints)
     integer,       intent(in), optional :: interval
-    integer :: j
+    integer :: j,n=1
+    
     if (print_titles) call write_str( 6, property, delim )
-    do j = 1, npoints
+    if (present(interval)) n=interval  
```

Comment:
This statement could have been placed outside the loop. The compiler will certainly do it during optimization, but it is better to do it explicitly for the sake of clarity.